Cargo.toml

```diff
@@ -16,9 +16,16 @@ orbfont = { version = "0.1.11", default-features = false, features = ["no-std"]
 plain = "0.2.3"
 redox_dmi = "0.1.5"
 redox_hwio = { version = "0.1.6", default-features = false }
+redox_uefi = "0.1.8"
```

Comment:
Should use `std::uefi` to avoid potential import conflicts (ref: 88ff9a046fcc).

Reply:
Will do